webdriver/tests/find_element_from_shadow_root/conftest.py

```diff
@@ -0,0 +1,71 @@
+@pytest.fixture
+def get_shadow_page(inline, shadow_content):
+    return inline("""
```

Comment:
I actually wonder how this can work with the current implementation. What you return here is a string as produced by `inline()`. In the tests you call `get_shadow_page(...)`, which has to fail on a string.

So what you want is to return an inner function; the same as what we do for most of the global fixtures. And only the inner method has the `shadow_content` argument.

This also applies to the other commands.

Reply:
Acknowledged. Will work on this.

Reply:
Took a stab at this in c9921ba. I've probably botched it, so do feel free to re-review.